gallery/fonts/flaticon/font/_flaticon.scss

```diff
@@ -0,0 +1,51 @@
+    /*
+    Flaticon icon font: Flaticon
+    Creation date: 07/12/2018 03:18
+    */
+
```

Comment:
@baamaboi remove these unnecessary comments. Do the same for all of the pages.

Reply:
This file no longer exists